lib/main/STM32_USB_OTG_Driver/src/usb_core.c

```diff
@@ -391,13 +392,13 @@ USB_OTG_STS USB_OTG_CoreInit(USB_OTG_CORE_HANDLE *pdev)
     /* Deactivate the power down*/
     gccfg.d32 = 0;
     gccfg.b.pwdn = 1;
+#ifdef VBUS_SENSING_ENABLED
```

Comment:
Not sure if we even want to do this, or if we should completely disable this, since we don't have anything in the firmware that can take advantage of VBUS_SENSING_ENABLED.
(Reason: There are probably a lot of users with their own targets in a fork out there, and if these are copied from an existing target there is a chance that it will contain VBUS_SENSING_ENABLED. This will give them a hard to reproduce / find bug.)